### ISSUE TITLE
editorconfig extension's fields cannot be set due to missing setters

### ISSUE THREAD

**Issue by Zougi** [ec4j/editorconfig-gradle-plugin#3]
Me again =D. I got the following error with bellow config block

```
editorconfig {
    includes = [ 'myfolder', 'secondfolder' ]
}
```

```
$ gradle editorconfigFormat

FAILURE: Build failed with an exception.

* Where:
Build file '/somefolder/build.gradle' line: 45

* What went wrong:
A problem occurred evaluating root project 'eos-jenkins'.
> Cannot set the value of read-only property 'includes' for object of type org.ec4j.gradle.EditorconfigExtension.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

Indeed it looks like EditorconfigExtension class has no public setter for "includes" and other properties

**Comment by ppalaga** [comment]
Sorry, I am new to Gradle so I hoped that Gradle sets the non-public fields using reflection much like Maven does. Of course I should have tested this. Let me add the setters.

**Comment by Zougi** [comment]
I believe in groovy we indeed don't have to create getters and setters, you could just remove the keyword private from the properties. I'm no expert neither =) I only started to work with groovy few months back and most of my knowledge is Jenkins DSLs oriented.

**Comment by ppalaga** [comment]
Fixed and released 0.0.2.
